presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -9215,4 +9215,50 @@ public void testDefaultDecimalLiteralSwitch()
         assertEquals(doubleColumnResult.getTypes().get(0), DOUBLE);
         assertEquals(doubleColumnResult.getMaterializedRows().get(0).getField(0), 1.0);
     }
+
+    @Test
+    public void testInnerJoinEmptyRightTable()
```

Comment:
s/Right/Build

I would also suggest you to name this test with feature you are testing. Now you described the query. Maybe something like `testInnerJoinShortCircuitOnEmptyBuildTable()`

Reply:
Updated, I like the suggestion about naming the test for the specific feature, the previous name was a bit generic.